pkgs/test_api/test/frontend/expect_async_test.dart

```diff
@@ -300,30 +300,12 @@ void main() {
     });
   });
 
-  group("with errors", () {
-    test("reports them to the current test", () async {
```

Comment:
It seems like we should restore this test (checking that async errors are reported)?

Reply:
I don't think we want them to be reported. It should be allowed to have some callback somewhere throw and not fail the test.

Reply:
Ah, I may have misunderstood.

I don't think that this test was useful for testing the behavior of async errors. That should be covered elsewhere.

Reply:
Ok, I don't have enough context here to know it just seems wrong on the surface to delete that test :man_shrugging: